pip/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+authors = ["Microsoft"]
+name = "pyqir_runner"
+version = "0.0.1"
```

Comment:
Fix version number here to match qir-runner versioning

Reply:
We were mistaken, version is coming from `pyproject.toml`

Reply:
Actually, this might not be needed, since the version number comes from the pyproject.toml. Maybe set to `0.0.0` to be explicit about it being unused?